src/pages/PublicHome.jsx

```diff
@@ -1,33 +1,55 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { Typography, Button } from '@mui/material';
+import { Button, Box } from '@mui/material';
 import ScholarshipsMadeSimple from '../components/ScholarshipsMadeSimple';
 import HomeSection from '../components/HomeSection';
-import community from '../img/community.png';
+import graduation from '../img/img2.svg';
+import searching from '../img/img3.svg';
 
 function PublicHome() {
   return (
     <>
-      <Typography variant="h2" align="center" gutterBottom>
-        Find Scholarships Today
-      </Typography>
-      <ScholarshipsMadeSimple />
       <HomeSection
-        alignItems="center"
-        direction="row-reverse"
-        title="For and by the Community"
-        description="Interested in helping students find scholarships? Join the family of community contributors, help find scholarships that are open to anyone regardless of citizen status."
+        direction="row"
+        title="Scholarships for everyone."
+        main
+        description="Find your dream scholarship together."
```

Comment:
FYI this wording was changed recently. I'm actually going to leave some comments on the Figma design for what our wording for a bunch of things could be.

Reply:
nice, I'll look out for those.